### PR TITLE
Moving the core of the Instant Articles functionality into base bulbs

### PR DESCRIPTION
**Pull request by spra85**

### CODE REVIEW

**Comment by benghaziboy** [comment]
@spra85 ?

**Comment by benghaziboy** [comment]
@mparent61 @MichaelButkovic @spra85

**Comment by benghaziboy** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;pQJVZJNB&#x2F;86-instant-articles"> Instant Articles</a>

**Comment by benghaziboy** [comment]
fellas

**Comment by MichaelButkovic** [comment]
👍

File: bulbs/instant_articles/templates/instant_article/_instant_article_ad.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% load pipeline core ads %}
+
```
Comment (benghaziboy) [inline bulbs/instant_articles/templates/instant_article/_instant_article_ad.html]: Need to call `{% targeting %}` here?

Comment (benghaziboy) [inline bulbs/instant_articles/templates/instant_article/_instant_article_ad.html]: Here: https://github.com/theonion/avclub/blob/b0c0901ce9cf12b43f6c8598525ee48f4d43c520/avclub/apps/instant_article/templates/instant_article/_instant_article_ad.html#L3

File: bulbs/instant_articles/views.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        context["search_url"] = self.request.build_absolute_uri("/")
+
+        for content in context["page_obj"].object_list:
+            content.feed_url = self.request.build_absolute_uri(content.get_absolute_url())
+            content_ctx = {
+                "content": content,
+                "absolute_uri": self.request.META.get('HTTP_HOST', None)
+            }
+            try:
+                content.instant_article_body = loader.render_to_string(
```
Comment (benghaziboy) [inline bulbs/instant_articles/views.py]: @spra85: Change this to `instant_article_html`

File: example/urls.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 urlpatterns = patterns("",
     url(r"^api/v1/", include("bulbs.api.urls")),  # noqa
     url(r"^", include("bulbs.poll.urls")),  # noqa
+    url(r"^", include("bulbs.instant_articles.urls")),  # noqa
```
Comment (benghaziboy) [inline example/urls.py]: In here twice... remove one

**Comment by benghaziboy** [comment]
@spra85 updated

File: bulbs/ads/views.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    req = RequestFactory().get(url)
+    kwargs['request'] = req
+    response = view(*args, **kwargs)
+    if response.status_code != 200:
+        return HttpResponseNotFound()
+
+    targeting = response.context_data.get('targeting', {})
+
+    if request.method == "POST":
```
Comment (spra85) [inline bulbs/ads/views.py]: What is this used for?  Curious when you'd ever make a `POST` request where this comes into play

**Comment by spra85** [comment]
👍

**Pull request merged**